executers/utils/utils.go

```diff
@@ -66,6 +107,13 @@ func GetCachePath() (string, error) {
 	}
 	return filepath.Join(goPath, "pkg", "mod", "cache", "download"), nil
 }
+func GetPackagePath() (string, error) {
```

Comment:
Missing new-line between the two functions.
Let's also add a description for this function and also for ```GetCachePath()```.